src/commcare_cloud/environment/schemas/app_processes.py

```diff
@@ -33,6 +33,7 @@ class AppProcessesConfig(jsonobject.JsonObject):
     formplayer_memory = MemorySpecProperty()
     http_proxy = IpAddressAndPortProperty()
     newrelic_djangoagent = jsonobject.BooleanProperty()
```

Comment:
This should not be necessary. Maybe we should change newrelic_django_prefix to just `django_command_prefix`? (it's only used on prod)